sampler.py

```diff
@@ -131,7 +132,7 @@ def initOptima(self, scores, energy, std_dev):
 
         # set initial values
         self.E0 = scores[1]  # initialize the 'best score' value
-        self.absMin = np.amin(self.E0)
+        self.absMin = np.amin(np.array(self.E0))
```

Comment:
why?

Reply:
Otherwise : 
TypeError: min() received an invalid combination of arguments - got (axis=NoneType, out=NoneType, ), but expected one of: